deck.json

```
@@ -1230,6 +1347,53 @@
             "tags": [
                 "health"
             ]
+        },
+        {
+            "__type__": "Note",
+            "data": "",
+            "fields": [
+                "How does the shortage of testing equipment affects the accuracy of official statistics in the COVID-19 pandemic?",
+                "Not enough supply means it is not possible to detect every cases of COVID-19.",
+                "Battegay, M., Kuehl, R., Tschudin-Sutter, S., Hirsch, H. H., Widmer, A. F., &amp; Neher, R. A. (2020). 2019-novel Coronavirus (2019-nCoV): estimating the case fatality rate–a word of caution.&nbsp;<i>Swiss medical weekly</i>,&nbsp;<i>150</i>(0506).<br><div><br></div>"
+            ],
+            "flags": 0,
+            "guid": "A7J^Yqc8?$",
+            "newlyAdded": false,
+            "note_model_uuid": "278ee8b3-6fe6-11ea-8f96-80e82c59ddd1",
+            "tags": [
+                "health"
+            ]
+        },
+        {
+            "__type__": "Note",
+            "data": "",
+            "fields": [
+                "How did South Korea flatten the curve?(tentative March 2020)",
+                "<div>- Quick emergency response.</div><div>- Test people in a safe manner as often and quickly as possible.</div><div>- Trace infected's journey, test anyone who came into contact, and quarantine the infected.</div><div>- Public trust and cooperation from the public.</div>",
+                "<div>\n<div>Fisher, Max, and Choe Sang-hun. “How South Korea Flattened the Curve.” <i>The New York Times</i>, The New York Times, 23 Mar. 2020, www.nytimes.com/2020/03/23/world/asia/coronavirus-south-korea-flatten-curve.html.&nbsp;</div></div>"
+            ],
+            "flags": 0,
+            "guid": "d@6:wMopFe",
+            "newlyAdded": false,
+            "note_model_uuid": "278ee8b3-6fe6-11ea-8f96-80e82c59ddd1",
+            "tags": [
+                "health"
+            ]
+        },
+        {
+            "__type__": "Note",
+            "data": "",
+            "fields": [
+                "How did South Korea flatten the curve?(tentative March 2020)<div><br></div><div><div>- {{c1::Quick emergency response.}}</div><div>- {{c2::Test people in a safe manner as often and quickly as possible.}}</div><div>- {{c3::Trace infected's journey, test anyone who came into contact, and quarantine the infected.}}</div><div>- {{c4::Public trust and cooperation from the public.}}</div></div>",
```

With the changes above, the cloze deletion could be simplified to
```suggestion
                "South Korea's initial success in flattening the curve: {{c1::speed of intervention}}, {{c2::broad and quick testing}}, {{c3::comprehensive tracing.}}, {{c4::public trust and cooperation}}</div></div>",
```